pkg/vectorize/sum/sum.go

```diff
@@ -94,3 +96,21 @@ func floatSumSels[T constraints.Float](xs []T, sels []int64) T {
 	}
 	return res
 }
+
+/*
```

Comment:
Please do not add these dead code.   If you really want to vectorize sum, do it completely in another PR.

Reply:
OK, sorry, I submitted it carelessly